code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -98,7 +98,7 @@
 //Pills
 /obj/item/weapon/reagent_containers/pill/antitox
 	name = "Anti-toxins pill"
-	desc = "Neutralizes many common toxins."
+	desc = "Neutralizes many common toxins. Contains 25 units of Dylovene."

```

Comment:
How are you able to tell this just by examining it? I believe this was originally intentionally obfuscated.

Reply:
Probably has a little '25' written on the back of the pill or something.

Reply:
Anewbe remains opposed to the labeled volumes, I have no preference.